packages/app/src/components/Bookmarks/BookmarkFolderItem.tsx

```diff
@@ -312,9 +305,9 @@ const BookmarkFolderItem: FC<BookmarkFolderItemProps> = (props: BookmarkFolderIt
             <div className='grw-foldertree-title-anchor pl-2' >
               <p className={'text-truncate m-auto '}>{name}</p>
             </div>
-            {hasChildren() && (
+            {bookmarks.length > 0 && (
               <div className="grw-foldertree-count-wrapper">
-                <CountBadge count={childCount} />
+                <CountBadge count={bookmarks.length} />
```

Comment:
# Change request
![image](https://user-images.githubusercontent.com/68407388/217527632-f3feba65-a8b8-427a-8de7-81c1fc5a53d6.png)

- Please count and display all pages under the folder.
- ex. 
  - `Parent` count 3 pages.
  - `Child01` count 2 pages.
  - `Child02` count 1 pages.